Update documentation to refer to c.JupyterHub.bind_url (#4042)
jupyterhub/jupyterhub: docs/source/getting-started/networking-basics.md

`jupyterhub_config.py`:

```python

> c.JupyterHub.bind_ip = '192.168.1.2:443'

Reviewer: it's `bind_url` here:

```suggestion
c.JupyterHub.bind_url = 'https://192.168.1.2:443'
```

Reviewer 2: agree with @minrk but probably understood that http:// or https:// based on no-ssl or ssl used.

most deployments of JupyterHub. However, more customized scenarios may need
additional networking details to be configured.

> Note that `c.JupyterHub.bind_ip` and `c.JupyterHub.port` are single values,
> not tuples or lists – JupyterHub listens to only a single IP address and

Reviewer: ```suggestion
Note that `c.JupyterHub.bind_url` is a single string,
not a tuple or list – JupyterHub listens to only a single IP address and
```

port.

**Deprecated in 0.9:** Alternatively, Proxy's main IP address and port can be set by placing the following lines in the configuration file:

```python
c.JupyterHub.ip = '192.168.1.2'
c.JupyterHub.port = 443
```

> We strongly recommend using the `bind_ip` setting instead of the deprecated
> settings, `ip` and `port`.

Reviewer: ```suggestion
We recommend using the `bind_url` setting instead of `ip` and `port`,
since `bind_url` can set ip, port, and base_url in one place.
```

Reviewer 2: agree with @minrk correction
